src/app/api/routes/media.py

```diff
@@ -1,9 +1,12 @@
-from typing import List
-from fastapi import APIRouter, Path
+
+from fastapi import APIRouter, Path, Security, File, UploadFile, HTTPException
 from app.api import crud
 from app.db import media
-from app.api.schemas import MediaOut, MediaIn
-
+from typing import List
+from app.api.schemas import MediaOut, MediaIn, MediaCreation, DeviceOut, BaseMedia
+from app.api.deps import get_current_device
+from app.services import bucket_service
+from datetime import datetime
```

Comment:
Better to separate imports of external libraries vs. current package. Perhaps fastapi, typing & datetime at the top?